src/patch/createPatchObject.ts

```diff
@@ -11,6 +11,8 @@ export interface Patch {
   start2: number
   length1: number
   length2: number
+  byteLength1: number
+  byteLength2: number
```

Comment:
What about `utf8Length` instead? `byteLength` isn't that precise since `length` today actually now is the byte length of the UCS2 encoding.